File: Data Structures & Algorithms/max-consecutive-ones/submission-7.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int max = 0;
        int current=0;
        for(int i = 0; i< nums.size(); i++){
            if (nums[i]==1){
              current++;
            }
            else{
                current = 0;
            }
            if (current>max){
                max=current;
            }
        }
        return max;
    }
};

File: Data Structures & Algorithms/max-water-container/submission-7.py
class Solution:
    def maxArea(self, heights: List[int]) -> int:
        max=0
        current=0
        left=0
        right=len(heights)-1
    
        while left < right:
            current= min(heights[left], heights[right]) * (right - left)
            if current > max:
                max=current
            if heights[right] > heights[left]:
                left +=1
            else:
                right-=1
        return max


        